src/pages/labelPage/LabelPage.jsx

```diff
@@ -1,9 +1,202 @@
+    }
+    if (filters.oldest) {
+      newData.sort((a, b) => getTime(a.createdDate) - getTime(b.createdDate));
+    }
+    if (filters.all) {
+      setFilteredData(newData);
+    } else {
+      const selectedLabels = labels.filter((item) => filters[item]);
+      newData = newData.filter((item) => selectedLabels.includes(item.label));
+      setFilteredData(newData);
+    }
+  }, [filters]);
+
   return (
     <section className="label">
-      <Label />
+      <div className="label-chips">
+        <button
+          type="button"
+          className={`btn ${
+            filters.all ? "btn-secondary" : "btn-secondary-outline"
+          } `}
+          onClick={() =>
+            setFilters({
+              ...filters,
+              all: !filters.all,
+              Personal: false,
+              Work: false,
+              Home: false,
+            })
+          }
+        >
+          All
+        </button>
+        <button
+          type="button"
+          className={`btn ${
+            filters.Home ? "btn-secondary" : "btn-secondary-outline"
+          } `}
+          onClick={() =>
+            setFilters({ ...filters, all: false, Home: !filters.Home })
+          }
+        >
+          Home
+        </button>
+        <button
+          type="button"
+          className={`btn ${
+            filters.Work ? "btn-secondary" : "btn-secondary-outline"
+          } `}
+          onClick={() =>
+            setFilters({ ...filters, all: false, Work: !filters.Work })
+          }
+        >
+          Work
+        </button>
+        <button
+          type="button"
+          className={`btn ${
+            filters.Personal ? "btn-secondary" : "btn-secondary-outline"
+          } `}
+          onClick={() =>
+            setFilters({ ...filters, all: false, Personal: !filters.Personal })
+          }
+        >
+          Personal
+        </button>
+      </div>
+      <div className="filters-container">
+        <div className="filter-section">
+          <h2 className="fs-m">Sort By Priority</h2>
+
+          <div className="filters">
+            <label className="fs-s form-label">
+              <input
+                type="radio"
+                className="form-radio"
+                name="sort"
+                value="high-to-low"
+                onChange={() =>
+                  setFilters({
+                    ...filters,
+                    highToLow: !filters.highToLow,
+                    lowToHigh: false,
+                  })
+                }
+              />
+              Price High to Low
+            </label>
+            <label className="fs-s form-label">
+              <input
+                type="radio"
+                className="form-radio"
+                name="sort"
+                value="low-to-high"
+                onChange={() =>
+                  setFilters({
+                    ...filters,
+                    lowToHigh: !filters.lowToHigh,
+                    highToLow: false,
```

Comment:
query - is the price filter required here ?

Reply:
Forgot to remove. Thanks for pointing!